frontend/src/metabase/dashboard/hooks/use-embed-theme.ts

```diff
@@ -7,7 +7,7 @@ import type { DisplayTheme } from "metabase/public/lib/types";
 
 import type { EmbedThemeControls } from "../types";
 
-export const useEmbedTheme = (): EmbedThemeControls => {
+export const useEmbedTheme = (initTheme: DisplayTheme): EmbedThemeControls => {
```

Comment:
This hook is now used in dashboard context, so it's not only about embedding anymore. we should rename it.

```suggestion
export const useDashboardTheme = (initTheme: DisplayTheme): EmbedThemeControls => {
```